tests/create_sequence_data.py

```diff
@@ -47,6 +57,7 @@ def create_sequence_data(
             fp = np.memmap(filename, dtype=data.dtype, mode="w+", shape=data.shape)
             fp[:] = data[:]
             fp.flush()  # Ensure data is written to disk
+            del fp
         meta["dtype"] = str(data.dtype)
 
         if shifts_per_signal:
```

Comment:
@reneburghardt @tomschewski could some of you please add for the future backlog - to create the tests where the phase shifts are above the target sampling rate (now they are all smaller)